src/main/java/com/customatics/leaptest_plugin/LeaptestJenkinsBridgeBuilder.java

```diff
@@ -142,6 +142,16 @@ public void perform(final Run<?,?> run, FilePath workspace, Launcher launcher, f
                 if(runId != null)
                 {
                     resultsMap.put(runId,leapWorkRun);
+                    boolean runIdReady = false;
+                    while ( !runIdReady ){
```

Comment:
@bicschneider , This is returning to API v2, when we had to wait till schedule is ready. API v3 was developed in such way to get all already executed cases, during schedule running